modules/core/test/test_mat.cpp

```diff
@@ -1719,17 +1719,23 @@ TEST(Mat, from_initializer_list)
 {
     Mat A({1.f, 2.f, 3.f});
     Mat_<float> B(3, 1); B << 1, 2, 3;
+    Mat_<float> C({1}, {1,2,3});
```

Comment:
This should not work (or better to compile). "Sizes" declares matrix with single element, but it is initialized with 3 values.

Reply:
I had tryed to use the following constructor for initialize.
->Mat (int rows, int cols, int type, const Scalar & s);
But, as you say, using the constructor below is correct.
->Mat (int ndims, const int * sizes, int type, const Scalar & s);
If I still do not understand what you say, I apologize.